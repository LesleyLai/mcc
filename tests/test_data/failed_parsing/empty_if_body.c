int main(void)
{
  if (0) else return 0;
}
