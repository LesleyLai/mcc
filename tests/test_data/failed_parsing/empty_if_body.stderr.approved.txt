{{filename}}:3:10: Error: Expect valid expression
3 |   if (0) else return 0;
  |          ^~~~

